Sources/ClientRuntime/Middleware/Attribute.swift

```diff
@@ -39,4 +39,8 @@ public struct Attributes {
     public mutating func remove<T>(key: AttributeKey<T>) {
         attributes.removeValue(forKey: key.name)
     }
+
+    public func getSize() -> Int {
+        return attributes.count
+    }
```

Comment:
The use case is to make sure there's at least one identity resolver configured on the service client. Checking non-nil isn't enough since default value is empty `Attributes`.

Reply:
But size could also be greater than zero due to any other non-identity related Attribute being set, right?

Reply:
Attributes is sort of intended to be a "grab bag" that can be used by unrelated parts of the codebase.  I wouldn't assume that the contents of Attributes will only be "x" at any time after its creation.

Reply:
In this case, I'm using an `Attributes` object to store only identity resolvers, which is used by getters and setters `getIdentityResolvers` and `withIdentityResolver`. So it's guaranteed to only have identity resolvers.

Reply:
What type is holding its own set of Attributes?

Reply:
So there's `HttpContext`, which has an `Attributes` object to keep track of all middleware context values. I'm storing identity resolvers in `HttpContext` by having the attribute key be of type `Attributes`, so nested `Attributes` basically.